IaC/ps.cd/JenkinsStack.yml_ps3

```diff
@@ -537,7 +537,7 @@ Resources:
                   wget -O /etc/yum.repos.d/jenkins.repo https://pkg.jenkins.io/redhat-stable/jenkins.repo
                   rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key
                   yum -y update --security
-                  yum -y install java-1.8.0-openjdk-1.8.0.191.b12 jenkins-2.176.2 git yum-cron aws-cli xfsprogs openresty
+                  yum -y install java-1.8.0-openjdk-1.8.0.191.b12 jenkins git yum-cron aws-cli xfsprogs openresty
```

Comment:
no way, please define the version

Reply:
it is my test jenkins, I would like to test this approach because now I am using 'stable' repo for jenkins and a few latest updates between stable versions  were fine. As you know for jenkins the main pain is plugins but not new stable version )))